qdrant-landing/content/blog/enterprise-vector-search-2025.md

```diff
@@ -0,0 +1,100 @@
+
+## Meet Qdrant's Enterprise Feature Suite
+
+Our enterprise solution handles scaling and securing vector search for production AI systems. Here's how:
+
+### Enterprise-Grade Security & Access Control
+
+**Cloud RBAC (Role-Based Access Control)** ***(Coming soon)*** → Define **granular permissions** to manage clusters, billing and Hybrid Cloud deployments.  
+[**Single Sign-On (SSO) Integration**](https://qdrant.tech/documentation/cloud/qdrant-cloud-setup/#enterprise-single-sign-on-sso) → Streamlined authentication via:  
+- Okta   
+- Entra ID (formerly Azure Active Directory)  
+- Azure Active Directory  
+- Active Directory/LDAP  
+- Azure Active Directory Native  
+- ADFS  
+- Google Workspace  
+- OpenID Connect  
+- PingFederate  
+- SAML  
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/EtUcA-MCZJM?si=RT2qMkA1pzbIWZA5" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+- [**Database API Keys with Granular Access Control**](https://qdrant.tech/documentation/cloud/authentication/) → Restrict API key access **per cluster and per collection**, enforce expirations, and revoke credentials instantly.
+
+**Why it matters:** Security is **baked in, not bolted on**. So you get **centralized control** over access management—simplifying compliance and reducing risk.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/3c-8tcBIVdQ?si=Kh_tcmTVGjPnq7Zh" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe> 
+
+### Full Observability with Monitoring
+
+* [**Monitoring**](https://qdrant.tech/documentation/guides/monitoring/) → **Real-time metrics via Prometheus/OpenMetrics**, integrating directly into your existing monitoring stack (Datadog, Grafana, etc.).
+
+**Why it matters:** Monitor performance, detect anomalies, and **stay ahead of potential issues**—all without custom instrumentation.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/pKPP-tL5_6w?si=ECiPZ-7D9usG-uIF" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+### API for simplified management
+
+* [**API**](https://qdrant.tech/documentation/cloud/authentication/) → Manage clusters, authentication methods, and cloud configurations. 
+
+**Why it matters:** Whether scaling manually or through automation, deploying and managing Qdrant is as easy as a few lines of code.
```

Comment:
Should we specifically mention, that we offer a terraform https://qdrant.tech/documentation/cloud-tools/terraform/ and pulumi https://qdrant.tech/documentation/cloud-tools/pulumi/ provider?